lib/otr.js

```diff
@@ -572,7 +574,7 @@
         if (this.REQUIRE_ENCRYPTION) {
           this.storedMgs.push({msg: msg, meta: meta})
           this.sendQueryMsg()
-          return
+          return promise
```

Comment:
This `promise` is never rejected or fulfilled, similarly all the cases below where the functions return early.

Reply:
This was by purpose, to resolve a Promise only if there were some data to send. But yes in this case `sendQueryMsg` should return a promise and also `notify` should return an rejected promise. Will update the pr accordingly.